testdata/public/package/example-1.0.0/index.json

```diff
@@ -79,6 +79,7 @@
       "name": "logs",
       "title": "Logs datasource",
       "description": "Datasource for your log files.",
+      "solution": "",
```

Comment:
I need to fix this on my end, should be skipped if empty.

Reply:
Not needed, I fixed it here.